pkg/occlient/occlient.go

```diff
@@ -3234,11 +3232,32 @@ func isSubDir(baseDir, otherDir string) bool {
 	return matches
 }
 
-// generateOwnerReference genertes an ownerReference which can then be set as
+// IsRouteSupported checks if route resource type is present on the cluster
+func (c *Client) IsRouteSupported() (bool, error) {
```

Comment:
is returning error really necessary here? we can return false for other cases.

Reply:
Actually I want to throw the error and stop the execution and inform the user straight away, otherwise the user might be confused on openshift clusters.